checker/src/visitors.rs

```diff
@@ -2284,6 +2286,32 @@ impl<'analysis, 'compilation, 'tcx, E> MirVisitor<'analysis, 'compilation, 'tcx,
         }
     }
 
+    fn try_extend_post_condition(&mut self, cond: &Rc<AbstractValue>) {
```

Comment:
Always prefix a function with a comment that summarizes what it does, a bit more expansively than what the name suggests.

Reply:
Done. Totally agree.